Gemfile

```diff
@@ -4,4 +4,5 @@ gem 'github-pages', group: :jekyll_plugins
 group :development, :test do
   gem 'rake'
   gem 'html-proofer'
+  gem 'jekyll-redirect-from'
```

Comment:
I believe this is already bundled with the `github-pages` gem

Reply:
I'm going off of this documentation: https://help.github.com/en/articles/redirects-on-github-pages

Reply:
Yup, I just don't think you need to add it to the gemfile, the rest looks good
> Make sure you're using the latest version of the GitHub Pages Gem, which contains these dependencies and plugins.
This also shouldn't hurt though unless the versions between the GitHub Pages gem and the other gem get out of sync. Don't think that should happen because there is no explicit version for either